components/static-pages/hiring.md

```diff
@@ -12,7 +12,7 @@ Join us at [Open Collective](https://opencollective.com/e2c) -- a tech platform
 <strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
 -->
 
-<em>February 15th 2022</em>. We're still hiring for the following roles:
+We're still hiring for the following roles:
```

Comment:
"we are still hiring" to me looks a bit weird like we have been trying to hire for a long time. 😉 

```suggestion
We're hiring for the following roles:
```